tests/test_identity.py

```diff
@@ -0,0 +1,115 @@
+import json
+import tempfile
+import time
+from pathlib import Path
+from typing import Generator
+
+import jwt
+import pytest
+
+from cryptography.hazmat.primitives.asymmetric import rsa
+from cryptography.hazmat.primitives import serialization
+from openeogeotrellis.identity import IDPTokenIssuer, IDP_TOKEN_ISSUER
+
+
+def test_idptokenissuer_should_not_be_instantiated_manually():
+    # WHEN someone wants to manually create an instance of the IDPTokenIssuer
+    # THEN they should get an exception
+    with pytest.raises(RuntimeError):
+        IDPTokenIssuer()
+
+
+def test_without_valid_config_token_should_just_be_none(monkeypatch):
+    # GIVEN no valid config
+    monkeypatch.setattr(IDP_TOKEN_ISSUER, '_IDP_DETAILS_FILE', Path("/tmp/file_that_do3s_not_exist.arr"))
```

Comment:
Isn't it possible to set `_IDP_DETAILS_FILE` from config (instead of from env var)? then it would be possible to override it in tests without monkeypatching

Reply:
I have moved it to config.